src/views/FindOrder.vue

```diff
@@ -396,6 +396,25 @@ ion-modal {
   --border-radius: 8px;
 }
 
+main > div:hover {
+  cursor: pointer;
+  background: rgb(240, 240, 240);
+}
+
+ion-item, ion-card{
+  --background: none;
+}
+
+@media (prefers-color-scheme: dark) {
+  main > div:hover {
+    background: rgb(63, 63, 63);
```

Comment:
use ion color variables.

Reply:
As per our discussion, I have removed hover effect changes and only added cursor pointer to order div.